src/pages/Home/Home.page.tsx

```
@@ -0,0 +1,16 @@
+import { ImageList } from '@containers';
+
+/**
+ * The Home page component.
+ *
+ * This component renders the application's home page.
+ *
+ * @returns {ReactElement} The Home component.
+ */
+export const Home = () => (
+    <Box p={4} width={'100%'}>
+        <ImageList></ImageList>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)